packages/base/src/util/removeLinkFromHead.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Removes a <link> tag in the <head> tag
+ * @param linkElement - reference to <link> tag
+ */
+const removeLinkFromHead = linkElement => {
```

Comment:
The reason we have `createLinkInHead` as a separate module is that we can pass attribute key/value pairs and a little more extra functionality. This however is just one line so IMO no need for a module, just call it manually.